stl/debugger/STL.natvis

```diff
@@ -1955,4 +2043,100 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     <Expand/>
   </Type>
 
+  <Type Name="std::formatter&lt;*,*&gt;">
+    <DisplayString>{_Specs._Dynamic_width_index}, {_Specs._Dynamic_precision_index}</DisplayString>
+    <Expand>
+      <Item Name="Width index">_Specs._Dynamic_width_index</Item>
+      <Item Name="Precision index">_Specs._Dynamic_precision_index</Item>
+    </Expand>

```

Comment:
Will this show the _Basic_format_specs members? those are basically as important as the _Dynamic_format_specs additional members.